### DISCUSSION TITLE
Difficulties Displaying Repos on Github

### DISCUSSION THREAD

**Discussion by jqma-usc** [community/community#43172] [New to GitHub]
Hi all,
I've participated in a few hackathons where I contributed (committed and pulled) from the repo that they own since I was a collaborator, but for some reason, I am not able to display these repos on my profile.
Has anyone else had the issue that they can assist me with?

**Comment by getlost01** [comment]
It's possible that the repository owner has not made the repository public, so it is not visible on your profile. Only public repositories are visible on user profiles by default. If you want the repositories to be visible on your profile, you can ask the repository owner to make the repositories public or you can create your own copies of the repositories (called "forks") and make your forks public. That way, you can retain your contributions while also making them visible on your profile.

**Comment by diva-21** [comment]
Make sure that the repo is public to everyone, then keep an instance of that by forking @jqma-usc
